### PR TITLE
[LinkedIn] Fix function name

### LINKED ISSUE #5158: [BUG] Linkedin Query Analytics Finder not working
The query_analytics_finder action is giving me this error, no matter what inputs I use: "this.generateQueryStringForParameter is not a function"

### PR DESCRIPTION
**Pull request by andrewjschuang**
Fixed and refactored `query-analytics-finder` legacy-converted action.

`@pipedream/platform/axios` doesn't handle well URLs with queries.
And [LinkedIn's API protocol](https://learn.microsoft.com/en-us/linkedin/shared/api-guide/concepts/protocol-version?context=linkedin%2Fconsumer%2Fcontext&view=li-lms-2023-01) `X-Restli-Protocol-Version: 2.0.0` changed a lot.

Resolves #5158.

### CODE REVIEW

File: components/linkedin/actions/query-analytics-finder/query-analytics-finder.mjs [hunk 1]
```diff
@@ -1,197 +1,129 @@
...
+    for (const facet of FACETS) {
+      if (this[facet]) {
+        query += `&${this.createListQuery(facet, this[facet])}`;
+      }
     }
 
-    if (dateRange.end.year) {
-      dateRangeQueryString += `&dateRange.end.year=${dateRange.end.year}`;
-    }
+    const url = `${constants.BASE_URL}${constants.VERSION_PATH}${PATH}${query}`;
```
[the remark this answers is not in the document]
Comment (andrewjschuang) [reply components/linkedin/actions/query-analytics-finder/query-analytics-finder.mjs:115]: Was testing and remembered that LinkedIn uses `WHATWG` standard for protocol `2.0.0`, that's why it doesn't work with `axios`

**Comment by andrewjschuang** [comment]
After merging with `master` some automatic lint fixes were made in the `Todoist` app, so I'm also bumping the version.

**Comment by DilanAthukorala** [comment]
Verification success, ready for release!

**Pull request merged**